### PR TITLE
feat(git) Remove all local branches that aren't in remote

### PR DESCRIPTION
**Pull request by guionardo**
## New workflow

Remove local git branches that aren't in remote.
After a merge, the work branch doesn't need to exists anymore.

### CODE REVIEW

File: specs/git/remove_local_branches_not_in_remote.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Remove all local branches that aren't in remote
+command: |-
+  "git branch --merged >/tmp/merged-branches && \
+   vi /tmp/merged-branches && \
+   xargs git branch -d </tmp/merged-branches"
+tags:
+  - git
+description: "Remove all local branches that doesn't exists in remote repository."
```
Comment (alokedesai) [inline specs/git/remove_local_branches_not_in_remote.yaml:9]: minor grammar nit: I think this should be "Remove all local branches that don't exist in a remote repository."

File: specs/git/remove_local_branches_not_in_remote.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: Remove all local branches that aren't in remote
```
Comment (alokedesai) [inline specs/git/remove_local_branches_not_in_remote.yaml:2]: maybe "Remove all local branches that aren't on the remote repository"?

**Review by alokedesai — COMMENTED** [review]
Looks great, thank you for contributing! Just one minor comment on the description!

File: specs/git/remove_local_branches_not_in_remote.yaml (hunk 2 above)
Comment (guionardo) [reply specs/git/remove_local_branches_not_in_remote.yaml]: Ok, updated.

**Review by guionardo — COMMENTED** [review]
Fixed

**Review by alokedesai — APPROVED** [review]
LGTM!

**Pull request merged**